### ISSUE TITLE
localeCompare does not return correct results.

### ISSUE THREAD

**Issue by maiwald** [ariya/phantomjs#14496]
1. Which version of PhantomJS are you using?
   2.1.1
2. What steps will reproduce the problem?
   See code below.
3. Which operating system are you using?
   Mac OS 10.11.6
4. Did you use binary PhantomJS or did you compile it from source?
   I used the homebrew compiled version
5. Please provide any additional information below.

This is a re-issue of https://github.com/ariya/phantomjs/issues/11063 since the problem is still present and nothing seems to happen on the old issue.

The following code returns faulty results in phantom.js compared to Chrome (52.0.2743.116) and Firefox (43.0.4):

``` javascript
'11'.localeCompare('100', 'de', {numeric: true}) // returns 1 instead of -1
'11 foo'.localeCompare('100 foo', 'de', {numeric: true}) // returns 1 instead of -1
'foo 11'.localeCompare('foo 100', 'de', {numeric: true}) // returns 1 instead of -1
'äbc'.localeCompare('ax', 'de', {numeric: true}) // returns 1 instead of -1
```

**Comment by Vitallium** [comment]
This will be fixed in the upcoming release 2.5.

**Comment by Vitallium** [comment]
Could you please try again with 2.5 binary? Thanks!

**Comment by antoine1fr** [comment]
> This will be fixed in the upcoming release 2.5.

When will the version 2.5 be released? I can't see any information related to 2.5 in the tags, branches and change log. There is no information in the change log regarding versions newer than 2.1.0. I'm using 2.1.16 (phantomjs-prebuilt@2.1.16) but am still experiencing the bug.
